### PR TITLE
fix: add missing options for team memberlist table

### PR DESCRIPTION
**Pull request by anikdhabal**
## What does this PR do?

Screenshot 2024-10-09 124205

## Mandatory Tasks (DO NOT REMOVE)

- [x] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [x] N/A-I have updated the developer docs in /docs if this PR makes changes that would require a [documentation change](https://cal.com/docs). If N/A, write N/A here and check the checkbox.
- [x] N/A-I confirm automated tests are in place that prove my fix is effective or that my feature works.

## How should this be tested?

- Are there environment variables that should be set?
- What are the minimal test data to have?
- What is expected (happy path) to have (input and output)?
- Any other important info that could help to test that PR

## Checklist

- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my changes generate no new warnings

### CODE REVIEW

File: packages/trpc/server/routers/viewer/bookings/addGuests.handler.ts [hunk 1]
```diff
@@ -44,7 +44,7 @@ export const addGuestsHandler = async ({ ctx, input }: AddGuestsOptions) => {
   if (!booking) throw new TRPCError({ code: "NOT_FOUND", message: "booking_not_found" });
 
   const isTeamAdminOrOwner =
-    (await isTeamAdmin(user.id, booking.eventType?.teamId ?? 0)) &&
+    (await isTeamAdmin(user.id, booking.eventType?.teamId ?? 0)) ||
```
Comment (anikdhabal) [inline packages/trpc/server/routers/viewer/bookings/addGuests.handler.ts:47]: Found this bug and fixed

**Review by sean-brydon — DISMISSED** [review]
Tested this! All seems to be working as expected! Nice job!

File: packages/features/ee/teams/components/MemberListItem.tsx [hunk 2]
```diff
@@ -111,8 +116,12 @@ export default function MemberListItem(props: Props) {
   const { data: session } = useSession();
   const utils = trpc.useUtils();
   const [state, dispatch] = useReducer(reducer, initialState);
+  const orgBranding = useOrgBranding();
+  const domain = orgBranding?.fullDomain ?? WEBAPP_URL;
```
Comment (anikdhabal) [inline packages/features/ee/teams/components/MemberListItem.tsx:120]: Updated for team inside the org

**Review by sean-brydon — APPROVED** [review]
LGTM - nice spot on the org url

**Pull request merged**